tests/ballerina-unit-test/src/test/resources/test-src/net/websub/hub/test_hub_startup.bal

```diff
@@ -22,8 +22,12 @@ function startupHub(int hubPort) returns websub:WebSubHub|websub:HubStartedUpErr
 }
 
 function stopHub(websub:WebSubHub|websub:HubStartedUpError hubStartUpResult) returns boolean {
-    websub:WebSubHub hub = hubStartUpResult but {
-        websub:HubStartedUpError hubStartedUpErr => hubStartedUpErr.startedUpHub
-    };
-    return hub.stop();
+    if (hubStartUpResult is websub:WebSubHub) {
+        websub:WebSubHub hub = hubStartUpResult;
+        return hub.stop();
```

Comment:
`hubStartUpResult` is a `websub:WebSubHub` object within this block right? 

Can't we just say
```ballerina
return hubStartUpResult.stop();
```